package.json

```diff
@@ -25,7 +25,7 @@
   "author": "Matt Lubner <matt@mattlubner.com>",
   "dependencies": {
     "javascript-barcode-reader": "^0.6.9",
-    "jimp": "^0.16.1",
+    "jimp": "^0.x",
```

Comment:
These are functionally equivalent _right now_, although using the `0.x` syntax is somewhat less secure, as it can result in a lesser version being installed under certain circumstances.

The latest version of `jimp` is `0.16.1`: https://www.npmjs.com/package/jimp